docker-app/qfieldcloud/core/admin.py

```diff
@@ -568,6 +568,27 @@ class ProjectFilesWidget(widgets.Input):
     template_name = "admin/project_files_widget.html"
 
 
+class OwnerTypeFilter(admin.SimpleListFilter):
+    # "owner__type" for the filter would suffice,
+    # But the filter title would then be "By type" in the UI,
+    # which is semantically unclear, hence the SimpleListFilter subclass
+    title = "owner type"
+    parameter_name = "owner_type"
+
+    def lookups(self, request, model_admin):
+        return [
+            (User.Type.PERSON, "Person"),
+            (User.Type.ORGANIZATION, "Organization"),
+            (User.Type.TEAM, "Team"),
+        ]
```

Comment:
```suggestion
        return (
            (User.Type.PERSON, _("Person")),
            (User.Type.ORGANIZATION, _("Organization")),
        )
```

Reply:
No, owner can only be Person or Organization:

In the core/models.py you can find:
```
class Project(models.Model):
    ...
    owner = models.ForeignKey(
        User,
        on_delete=models.CASCADE,
        related_name="projects",
        limit_choices_to=models.Q(type__in=[User.Type.PERSON, User.Type.ORGANIZATION]),
        help_text=_(
            "The project owner can be either you or any of the organization you are member of."
        ),
    )
    ...
```

Reply:
done